qpc_tools/utils.py

```diff
@@ -161,12 +162,12 @@ def get_password(args_dictionary):
     :param args_dictionary: the dictionary containing the args and values
     :returns: the dictionary with updated passwords
     """
-    arg_prompt = {
-        'rh_registry_username': 'Enter registry.redhat.io username:',
-        'rh_registry_password': 'Enter registry.redhat.io password:',
-        'server_password': 'Enter server password:',
-        'db_password': 'Enter database password:'
-    }
+    arg_prompt = OrderedDict([
+        ('rh_registry_username', 'Enter registry.redhat.io username:'),
```

Comment:
I think usually there is one space after prompts, for example when you do a yum install:
```
Is this ok [y/N]: codyNOTcory
```
However, all of our prompts do not have that.

Reply:
Added a space. Good idea 🙂

Reply:
👍